### PR TITLE
Migrate pkg/install_tab.mustache

### PR DESCRIPTION
**Pull request by isoos**
- removed the numbered section header from before "Install it" (see [pana install page](https://pub.dev/packages/pana/install))
- otherwise no changes, maybe a bit simpler logic

### CODE REVIEW

File: app/lib/frontend/templates/views/pkg/install_tab.dart [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    d.h2(text: 'Use this package as an executable'),
+    d.h3(text: 'Install it'),
+    d.p(text: 'You can install the package from the command line:'),
+    d.pre(
+      attributes: {'data-text-to-copy': 'dart pub global activate $package'},
+      child: d.code(
+        classes: ['language-shell'],
+        child: d.strong(text: 'dart pub global activate $package'),
+      ),
+    ),
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/install_tab.dart:49]: Why don't we make a helper method for `codeSnippet(language: ..., text: , child: , children: )`.

Then we don't have to write `pre` and `code`, and all the attributes.. ideally, we don't even need to supply `data-text-to-copy` if `text` is given..

File: app/lib/frontend/templates/views/pkg/install_tab.dart [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    d.h2(text: 'Use this package as a library'),
+    d.h3(text: 'Depend on it'),
+    d.p(text: 'Run this command:'),
+    if (usePubGet) _pubAddNode('Dart', 'dart $pubAddCommand'),
+    if (useFlutterPackagesGet) _pubAddNode('Flutter', 'flutter $pubAddCommand'),
+    d.p(children: [
+      d.text('This will add a line like this to your package\'s '
+          'pubspec.yaml (and run an implicit '),
+      d.code(
+          classes: ['language-shell'],
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/install_tab.dart:95]: Does this have any effect, when it's not inside a pre?

**Review by jonasfj — APPROVED** [review]
Consider if we want a `codeSnippet` helper, or we can refactor to create such a function later.

File: app/lib/frontend/templates/views/pkg/install_tab.dart (hunk 2 above)
Comment (isoos) [reply app/lib/frontend/templates/views/pkg/install_tab.dart:95]: Looks like hljs doesn't extend it with its own attributes, so I'll just remove it.

**Pull request merged**